Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputSystemProfile.cs

```diff
@@ -74,6 +76,24 @@ public MixedRealityPointerProfile PointerProfile
         /// </summary>
         public IMixedRealitySpeechSystem SpeechSystem => speechSystem ?? (speechSystem = MixedRealityToolkit.Instance.GetService<IMixedRealitySpeechSystem>());
 
+
+        /// <summary>
+        /// The list of cultures where speech recognition is supported.  Cultures added as needed
+        /// </summary>
+        private List<CultureInfo> supportedVoiceCultures = new List<CultureInfo>
+        {
+            new CultureInfo("en-US")
+        };
+
+        /// <summary>
+        /// REturns whether speech is supported for the current language/culture or not
+        /// </summary>
+        public bool IsSpeechSupported
```

Comment:
I don't think the name of this is correct for what it's trying to signify.

Also this seems to conflict with the existing ways to check if the system is enabled, thoughts @StephenHodgson

Reply:
The main intention with this (and why it wasn't configurable in the editor) was that it was meant to be a placeholder for indicating that a specific culture could be supported by the OS platform, rather than by the App.  The developer (I believe) has other tools of indicating supported languages through their packaging/deployment rather than runtime.  I'm going to revert this portion of the change for now so that we can have a proper design review.